src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,117 @@
+package frc.team2412.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    // Constants
+
+    public static class IntakeConstants {
+        public static String teamColor = "placeholder";
+        public static final double INTAKE_SPEED = 1;
+
```

Comment:
Best not to run the motor at full speed, as it uses a lot of current, generates a lot of heat, and doesn't run as efficiently.  See the Hardware User Manual link on the [Falcon 500 product page](https://store.ctr-electronics.com/falcon-500-powered-by-talon-fx/) for full performance data. Depending on the output gear ratios, somewhere between 0.5 and 0.8 is usually good.